pkg/metrics/collector/collector.go

```diff
@@ -30,6 +30,10 @@ func NewFsMetricsCollector(m *types.FsMetrics, imageRef string) *FsMetricsCollec
 	return &FsMetricsCollector{m, imageRef}
 }
 
+func NewDaemonInfoCollector(version *types.BuildTimeInfo, countType NydusDaemonCountType) *DaemonInfoCollector {
```

Comment:
How about redesigning the interface to `func NewDaemonInfoCollector(version *types.BuildTimeInfo, value float64) *DaemonInfoCollector`, where value can be negative

Then we can directly call `Add()` of gauge type

Reply:
It is more reasonable. I have finished it.